tests/suites/test_suite_mpi.function

```diff
@@ -1,11 +1,26 @@
 /* BEGIN_HEADER */
 #include "mbedtls/bignum.h"
 #include "mbedtls/entropy.h"
+#include "bignum_core.h"
+#include "bignum_mod.h"
+#include "bignum_mod_raw.h"
+#include "constant_time_internal.h"
+#include "test/constant_flow.h"
 
 #if MBEDTLS_MPI_MAX_BITS > 792
 #define MPI_MAX_BITS_LARGER_THAN_792
 #endif
 
+#define ciL    (sizeof(mbedtls_mpi_uint))   /* chars in limb  */
+#define biL    (ciL << 3)                   /* bits  in limb  */
```

Comment:
There two, and biH, should probably be in `bignum_internal.h`, since - even in development - they're already in 2 source files

Reply:
If we put them into any of the headers, we need to add an `MBEDTLS_` prefix to make `check_names` happy. Duplicating them seemed a better choice.

Reply:
Hmm, duplicating is nearly always a bad idea. `check_names` really shouldn't care about an `_internal.h` header file, surely?

Reply:
`check_names` evolved from back when there were no internal headers. Nowadays, we could change it to allow non-namespaced macro names in internal headers. (Identifiers with external linkage have to be namespaced regardless of whether they're declared in public headers.) Until relatively recently `check_names` was a very ad hoc shell script that was hard to modify, but we've rewritten it in python and it should be an easy patch now.